src/MagnumPlugins/UfbxImporter/Test/UfbxImporterTest.cpp

```diff
@@ -2802,6 +2802,7 @@ void UfbxImporterTest::animationInterpolation() {
 
 void UfbxImporterTest::animationRotationPivot() {
     auto&& data = ResampleRotationData[testCaseInstanceId()];
+    setTestCaseDescription(Utility::format("resampleRotation={}", data.resampleRotation ? "true" : "false"));
```

Comment:
Not sure if I'm doing something wrong but passing the `bool` directly fails on MSVC with an error about missing `Corrade::Utility::Implementation::Formatter<bool,void>`:

```
Corrade/Utility/Format.h(310,61): error C2027: use of undefined type 'Corrade::Utility::Implementation::Formatter<bool,void>'
Corrade/Utility/Format.h(310): message : see declaration of 'Corrade::Utility::Implementation::Formatter<bool,void>'
Corrade/Utility/Format.h(367): message : see reference to function template instantiation 'Corrade::Utility::Implementation::BufferFormatter::BufferFormatter<bool>(const T &)' being compiled
with [ T=bool ]
Corrade/Utility/Format.h(354): message : see reference to function template instantiation 'size_t Corrade::Utility::formatInto<bool>(const Corrade::Containers::MutableStringView &,const char *,const bool &)' being compiled
MagnumPlugins/UfbxImporter/Test/UfbxImporterTest.cpp(2942): message : see reference to function template instantiation 'String Corrade::Utility::format<bool,Corrade::Containers::String,Corrade::Containers::MutableStringView>(const char *,const bool &)' being compiled
with [ String=Corrade::Containers::String ]
Corrade/Utility/Format.h(310,1): error C2672: 'format': no matching overloaded function found
```

Reply:
Huh. It's completely possible that I never needed to print a bool yet, so the support for them is unimplemented. I'll check :)

I'd do just this though, no need to get complicated:

```cpp
setTestCaseDescription(data.resampleRotation ? "resample rotation" : "");
```

Reply:
Ah, alright, and `true` is default so my suggestion was stupid. Keep it like this, then :)